dashboard/test/ui/features/hour_of_code/hour_of_code_finish.feature

```diff
@@ -17,20 +17,23 @@ Scenario: Completing Minecraft HoC should go to certificate page and generate a
   And I press "button:contains(Submit)" using jQuery
   And I wait to see element with ID "uitest-thanks"
 
+@eyes
```

Comment:
Learning Q: is this how you create an Eyes test? 🙀 If yes, is it the only line you need to add before the rest of the test (`Scenario: Flappy…`), or are there other steps?

Reply:
You need the `@eyes` and then the lines involving `When I open my eyes...`, `And I see no difference...`, and `And I close my eyes`

Reply:
Oooo cool, thanks!